src/components/molecules/ChatMessageBox/ChatMessageBox.scss

```diff
@@ -0,0 +1,44 @@
+@import "scss/constants";
+
+.ChatMessageBox {
+  &__form {
+    display: flex;
+    align-items: center;
+    padding-bottom: 5px;
+  }
+
+  &__input {
+    // @debt Workaround to avoid styles set by src/scss/global.scss (fix this when when clean that up)
+    all: unset;
+    outline: none;
+    background-color: $saturated-black;
+    flex: 1;
+    cursor: pointer;
+    font-size: 0.9em;
+    padding: 12px 12px;
+    border-radius: 28px;
+    color: $white;
+  }
```

Comment:
We might be able to remove this entirely when we use our generic `InputField` component here rather than a raw `input` element.